internal/workflow_testsuite.go

```diff
@@ -32,6 +32,8 @@ import (
 	"testing"
 	"time"
 
+	"go.temporal.io/api/failure/v1"
```

Comment:
There is our internal convention use `pb` suffix for all api(=proto) imports. So this should be `failurepb`. You have it in other places. Please fix it also here too and everywhere else (if any).

Reply:
Will do